rbtcwrapperproxy/contracts/RBTCWrapperProxy.sol

```diff
@@ -19,6 +20,7 @@ contract RBTCWrapperProxy is ContractRegistryClient {
 
     address public wrbtcTokenAddress;
     address public sovrynSwapNetworkAddress;
```

Comment:
there is `ContractRegistryClient.addressOf(SOVRYNSWAP_NETWORK)`, shouldn't it be used intead?